MEMBER_EXPECTATIONS.md

```diff
@@ -1,31 +1,58 @@
-# Member Expectations
+# Working Group Member Expectations
 
-All participants in the OpenJS Foundation projects and groups must follow the [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CODE_OF_CONDUCT.md).
-There are further expectations for members who represent the Standards Working Group (hereafter called representatives).
+All participants in OpenJS Foundation projects and Working Groups must follow the [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CODE_OF_CONDUCT.md). Standards Working Group Members are expected to participate in discussions related to OpenJS Foundation Projects, open source software, and the standards and specifications that bear on them. They are also expected educate and mentor newcomers to the field of standardization, lending support and advice to projects pursuing specification development. Finally, they are expected to help export OpenJS Foundation values to the standards communities they work with in order to make standards bodies nicer places to do work.  
 
-It is understood that representatives may form opinions based on the needs of their employers' or their personal projects. While representing
-the Standards Working Group, however, they should decide/discuss/promote based on the interest of the Standards Working Group.
-If there is a conflict (due to personal or company specific projects), then the representative should be explicit
-about the hat they are wearing while deciding/discussing/promoting the project.
+There are further expectations for [OpenJS Foundation Representatives](https://github.com/openjs-foundation/standards/blob/main/MEMBER_REPRESENTATION.md) to standards and liaison organizations (hereafter called Representatives).
 
-Since many members are often a part of one or more projects in the foundation, feel free to discuss it with the members
-if there is a potential conflict while representing. The representative can raise an issue and discuss them during
-the Standards Working Group meeting.
+## Expectations for Representatives
```

Comment:
Could we consider starting off by saying that consensus of the Standards WG allows a Representative to represent the position of OpenJSF as a whole on a particular topic? And that conversely, unless Rep has said consensus, you're not in a capacity to represent OpenJSF's position on that topic?

Reply:
tl;dr: this would effectively prevent participation

This would be a very high bar to meet. Organization representatives are practically _always_ representing their organization's positions on topics.  For example, where participants in W3C working groups or delegates in Ecma committees represent an organization, those are the terms of their participation.  Strictly speaking, they do not participate in a personal capacity at all, so, at least in matters pertaining to normative topics, they are unable to represent a personal position, and as such are exclusively representing the position of their organization.

The OpenJS standards group does not have time to get consensus on all topics that may come up at standards bodies, and would very often find it difficult to even get a quorum of members that could speak to a given proposal.  We also typically only know topics one or two weeks in advance, if that.

Reply:
having further read some of your comments further below, it sounds like you may be referring to things at, for example, the AC (W3C) or GA (Ecma) level rather than at the WG or TC level.  I think this section on member representation of the OpenJS standards group refers to participation in WGs/TCs.

in any case, it's good to be on the same page one way or the other